Dockerfile

```diff
@@ -9,7 +9,7 @@ VOLUME ["/var/lib/odoo"]
 EXPOSE 8069 8072
 
 # Subimage triggers
-ONBUILD ENTRYPOINT ["/opt/odoo/common/entrypoint.sh"]
+ONBUILD ENTRYPOINT ["/opt/odoo/common/entrypoint"]
```

Comment:
OK, leave this one without extension, but please add it to scripts under `common/build.d`. It's not just humanization, it helps text editors too.